### PR TITLE
Plant release notes skeleton for next release

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

File: docs/cuda-miner.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The release packages do not ship with a CUDA miner, but you can build it yoursel
 - Epoch node configuration
 
 The documentation below is tested on:
-- Epoch version 0.19.0
```
Comment (lucafavatella) [inline docs/cuda-miner.md]: @dincho I took the liberty of bumping this too.

**Comment by lucafavatella** [comment]
Labelling as WIP as not meant to be merged yet - after release.

File: docs/cuda-miner.md (hunk 1 above)
Comment (dincho) [reply docs/cuda-miner.md]: well, this will not be technically correct, because no one would test it on each release

Comment (velzevur) [reply docs/cuda-miner.md]: I assume CI tests are still tests. Code and documentation shall be corresponding to each other, so if we've missed a thingy - that would certainly be a bug. Iagree on the bumping.

Comment (dincho) [reply docs/cuda-miner.md]: True, but the CI does not test CUDA install and deployment, neither we deploy GPU miners on out testnet (yet?).

I see this line more like disclaimer, thus my comment.

**Pull request merged**